OneSignalSDK/unittest/src/test/java/com/test/onesignal/MainOneSignalClassRunner.java

```diff
@@ -1168,6 +1167,19 @@ public void testDisableOpeningLauncherActivityOnNotificationOpen() throws Except
       assertEquals("Test Msg", lastNotificationOpenedBody);
    }
 
+   @Test
+   public void testLaunchUrlSuppressTrue() throws Exception {
+      // Add the 'com.onesignal.suppressLaunchURLs' as 'true' meta-data tag
+      OneSignalShadowPackageManager.addManifestMetaData("com.onesignal.suppressLaunchURLs", "true");
```

Comment:
can we have the case for false?

Reply:
Yes, I can add one, but the default value is false, wouldn't this value be inferred in the test that opens the URL (testOpeningLaunchUrl())?

Reply:
Yeah but still we should check all scenarios. It will be 
`OneSignalShadowPackageManager.addManifestMetaData("com.onesignal.suppressLaunchURLs", "false");`